ui/raidboss/data/02-arr/raid/t9.js

```diff
@@ -464,35 +462,41 @@
         'Iron Chariot': 'Eiserner Streitwagen',
         'Lunar Dynamo': 'Lunarer Dynamo',
         'Megaflare': 'Megaflare',
-        '(?<! )Meteor(?! Stream)': 'Meteor',
         'Meteor Stream': 'Meteorflug',
         'Raven Dive': 'Bahamuts Schwinge',
         'Ravensbeak': 'Bradamante',
         'Ravensclaw': 'Silberklauen',
         'Stardust': 'Sternenstaub',
         'Super Nova': 'Supernova',
         'Thermionic Beam': 'Thermionischer Strahl',
+        '\\(East\\)': '(Osten)',
+        '\\(In\\)': '(Rein)',
+        '\\(North\\)': '(Norden)',
+        '\\(Out\\)': '(Raus)',
+        '\\(South\\)': '(Süden)',
+      },
+      '~effectNames': {
+        'Garrote Twist': 'Leicht fixierbar',
+        'Raven Blight': 'Pestschwinge',
       },
     },
     {
       'locale': 'fr',
       'replaceSync': {
         'Astral Debris': 'Débris astral',
         'Dalamud Fragment': 'Débris de Dalamud',
+        'Dalamud Spawn': 'golem de Dalamud',
```

Comment:
Golem de Dalamud
Thank you.

Reply:
Its sync, so no need to update it.
Only replacementText need to be updated